RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -705,6 +715,47 @@ HcalNoiseInfoProducer::fillcalotwrs(edm::Event& iEvent, const edm::EventSetup& i
   return;
 }
 
+// ------------ fill the summary info from jets
+void
+HcalNoiseInfoProducer::filljetinfo(edm::Event& iEvent, const edm::EventSetup& iSetup, HcalNoiseSummary& summary) const
+{
+    bool goodJetFoundInLowBVRegion = false; // checks whether a jet is in
+                                            // a low BV region, where false
+                                            // noise flagging rate is higher.
+    if (!jetCollName_.empty())
+    {
+        edm::Handle<reco::PFJetCollection> pfjet_h;
+        iEvent.getByToken(jet_token_, pfjet_h);
+
+        if (pfjet_h.isValid())
```

Comment:
wouldn't it make more sense to let the framework throw if the jet collection is not available? Otherwise, the result seems inconsistent